app/models/miq_worker.rb

```diff
@@ -372,7 +372,7 @@ def self.supports_container?
   end
 
   def self.containerized_worker?
-    # un-rearch containers for Hammer
+    # un-rearch containers for Ivanchuk
```

Comment:
Not sure about this one... pods `master` isn't Ivanchuk?

Reply:
Not sure it matters. We could probably just remove the release name from this comment as it's unlikely that we'll get to containerized workers in an actual release.